docusaurus/docs/publish-a-plugin/sign-a-plugin.md

```diff
@@ -1,74 +1,88 @@
 ---
 id: sign-a-plugin
 title: Sign a plugin
-sidebar_position: 2
+sidebar_position: 4
+description: "How to sign a Grafana plugin."
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - sign plugin
+  - signing plugin
 ---
 
-<head>
-  <meta name="robots" content="noindex" />
-</head>
+# Sign a plugin
 
-Grafana requires all plugins to be signed so that we can verify their authenticity with signature verification.
+Grafana requires all plugins to be signed so that we can verify their authenticity with [signature verification](https://grafana.com/docs/grafana/latest/administration/plugin-management#plugin-signatures).
```

Comment:
Can be a follow up. But theres unnecessary duplication in this and the next sentence

Reply:
Fixed.